tests/test_wave_01.py

```diff
@@ -4,7 +4,7 @@
 from viewing_party.party import *
 from tests.test_constants import *
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
```

Comment:
Minor style preference: I would just delete these skip decorators rather than comment them out. Just helps keep the code cleaner.

There's a few other skip decorators that are like this, but I'll only say it once.

Having talked to other students, they ran into issues trying to delete rather than comment, but I tried it myself and it seemed to work OK.